0053-bundle-format/README.md

```diff
@@ -0,0 +1,483 @@
+
+**Solution**: Use online profile for signing and online verify with internal URLs or custom trusted root
+
+```bash
+
+# Sign with private infrastructure
+zarf package sign zarf-package-app-amd64.tar.zst \
+  --oidc-issuer https://oauth2.internal.company.com/auth \
+  --rekor-url https://rekor.internal.company.com \
+  --profile online
+
+# Verify with private infrastructure in an "online" profile
+Zarf package verify zarf-package-app-amd64.tar.zst \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --profile online
+
+# or verify in the default offline stance with a custom trusted-root
+# Create custom trusted root
+cosign trusted-root create \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --output custom_trusted_root.json
+
+# Verify with custom trusted root
+zarf package verify zarf-package-app-amd64.tar.zst \
+  --trusted-root custom_trusted_root.json
+```
+
+### Risks and Mitigations
+
+#### Risk 1: Embedded Trusted Root Expiration
+
+**Risk**: If embedded trusted root is not updated, verification may fail due to expired keys.
+
+**Mitigation**:
+- Explicit support for including updated Trusted Roots from the command line
+- Pre-release checklist includes trusted root update verification
+- Monitor Sigstore for key rotations
+- Renovate monitoring for updates to the Trusted Root repository
+- Documentation on manual trusted root updates
+
+#### Risk 2: Bundle Format Incomplete in Offline Mode
+
+**Risk**: Sigstore bundle may require fields that cannot be populated offline.
+
+**Mitigation**:
+- Validate minimal bundle structure works with cosign verification given offline keypair signing and verification
+- Document minimal vs. complete bundle differences
+
+#### Risk 3: Profile Confusion and Misconfiguration
+
+**Risk**: Users may choose wrong profile or misconfigure connectivity settings.
+
+**Mitigation**:
+- Default to offline profile (safest, most aligned with Zarf mission)
+- Validation of profile compatibility with flags
+- Examples in documentation for each profile
+
+## Design Details
+
+### Connectivity Profiles
+
+Connectivity Profiles will allow Zarf to abstract orchestrating what configurations are made to enable/disable network connectivity.
+
+#### Offline Profile (Default)
+
+**Purpose**: Complete offline operation without any network dependencies.
+
+**Signing Behavior**:
+- Supports keypair-based signing only
+- Supports cloud KMS if accessible without internet (e.g., AWS KMS via VPC endpoint)
+- Does NOT upload to transparency log (`tlogUpload = false`) - therefor does NOT contact Fulcio
+- Does NOT contact Fulcio (no keyless signing support)
+
+**Verification Behavior**:
+- Uses embedded Sigstore trusted root (fetched at Zarf build time via The Update Framework (TUF))
```

Comment:
To clarify, this is the trusted root embedded in to the Zarf binary itself, not when the package is built, do I have that right?